test/integration/api/allocation/correct_window_values_test.go

```diff
@@ -0,0 +1,113 @@
+package allocation
+
+// Tests AllocationAPI returns a valid Window.start and Window.end 
+// This is PoC test, there are lot of cases like "lastweek", "30m" and Unix timestamp values
+// Since I don't find the end date calculation consistent in the response, I have chosen to hardcode based on simple examples.
+
+// Test Case Pass Criteria
+// If Window.Start and Window.End correspond to the same time window as Input Window Argument
+
+import (
+	"time"
+	"testing"
+	"github.com/opencost/opencost-integration-tests/pkg/api"
+)
+
+//Calculate time window given a time string
+func calculateTimeWindow(window string, apiObj api.Window) (bool, api.Window) {

```

Comment:
Don't want to be too strict on these PRs, but generally you want to have the return types be `([result], [success])` -- you can ignore this, just wanted to point it out :)